ocaml/xcp-rrdd/bin/rrdp-cpu/rrdp_cpu.ml

```diff
@@ -106,6 +107,23 @@ let dss_vcpus xc doms =
                     blocked"
                  ~ty:Rrd.Derive ~default:false ~min:0.0 ()
              )
+          :: ( Rrd.VM uuid
+             , Ds.ds_make ~name:"runstate_runnable" ~units:"(fraction)"
+                 ~value:
+                   (Rrd.VT_Float
+                      (Int64.to_float
+                         (ri.Xenctrl.time1
+                         ++ ri.Xenctrl.time2
+                         ++ ri.Xenctrl.time5
+                         )
+                      /. 1.0e9
```

Comment:
Just out of curiosity, what does this `1.0e9` stand for? The interval of sampling which is 5 seconds?

Reply:
That's the conversion from nanoseconds to seconds

Reply:
Can we add comments what these `timeN` are? It's bad enough that we don't have better names in Xenctrl.